King-Benx/android-level-up
# ft #156005300: Integrate fastlane

#### What does this PR do?
This PR integrates fastlane into our android application

#### Description of Task to be completed?
Initially, there was no integration of tools such as slack to be able to monitor our project.
This PR delivers fastlane integration that enables us send notifications to slack.

#### SCREENSHOTS
<img width="1440" alt="screen shot 2018-08-28 at 12 52 53" src="https://user-images.githubusercontent.com/17830204/44715909-a648d080-aac1-11e8-9fc7-147f86b96df3.png">

#### How should this be manually tested?
1. Clone the repository onto your local machine
2. Set the following environment variables in your terminal
```
	export LC_ALL=en_US.UTF-8
	export LANG=en_US.UTF-8
```

3. Open android-level-up workspace on slack.
4. Run `fastlane test` in your terminal
5. Run `fastlane mock` in your terminal to upload apk to slack
6. Check converge-codelab-qa

#### Background context
NONE

#### What are the relevant pivotal tracker stories?
[Delivers #156005300]
